Missing Symbols

```
nico@nico-debian:~/projects/msemu$ cd src/z80em
nico@nico-debian:~/projects/msemu/src/z80em$ make

# ... blah blah blah ...

nico@nico-debian:~/projects/msemu/src/z80em$ cd ..
nico@nico-debian:~/projects/msemu/src$ make
gcc -Os -Wall -DLSB_FIRST -DX86_ASM -DDEBUG -c msemu.c -o msemu.o
In file included from msemu.c:17:0:
flashops.h:10:13: warning: inline function ‘readDataflash’ declared but never defined
 inline byte readDataflash(unsigned int translated_addr);
             ^~~~~~~~~~~~~
flashops.h:9:13: warning: inline function ‘writeCodeFlash’ declared but never defined
 inline void writeCodeFlash(uint32_t translated_addr);
             ^~~~~~~~~~~~~~
flashops.h:8:16: warning: inline function ‘readCodeFlash’ declared but never defined
 inline uint8_t readCodeFlash(uint32_t translated_addr);
                ^~~~~~~~~~~~~
gcc -Os -Wall -DLSB_FIRST -DX86_ASM -DDEBUG -c flashops.c -o flashops.o
gcc -Os -Wall -DLSB_FIRST -DX86_ASM -DDEBUG z80em/Z80.o z80em/Z80Debug.o flashops.o  msemu.o  -o msemu  -lSDLmain  -lSDL  -lSDL_gfx 
msemu.o: In function `Z80_RDMEM':
msemu.c:(.text+0x5ed): undefined reference to `readCodeFlash'
msemu.c:(.text+0x647): undefined reference to `readDataflash'
collect2: error: ld returned 1 exit status
makefile:24: recipe for target 'msemu.exe' failed
make: *** [msemu.exe] Error 1
```

--- a/src/flashops.h
+++ b/src/flashops.h
@@ -5,9 +5,9 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-inline uint8_t readCodeFlash(uint32_t translated_addr);
-inline void writeCodeFlash(uint32_t translated_addr);
-inline byte readDataflash(unsigned int translated_addr);
+uint8_t readCodeFlash(uint32_t translated_addr);
+void writeCodeFlash(uint32_t translated_addr);
+byte readDataflash(unsigned int translated_addr);
 int8_t writeDataflash(unsigned int translated_addr, byte val);
 int flashtobuf(uint8_t *buf, const char *file_path, ssize_t sz);
 int buftoflash(uint8_t *buf, const char *file_path, ssize_t sz);

